Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tikv"
-version = "0.0.1"
+version = "0.9.0-pre-ga"
```

Comment:
Below 1.0.0, we may also meet incompatibility, E.g, latter 0.9.0 for real dedicated raft RocksDB.

So I prefer using 0.8.0 here, not 0.9.0.